ci/deps/actions-310.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   - pytz
 
   # optional dependencies
+  - aiobotocore<2.0.0
```

Comment:
e.g. here

Reply:
I mentioned this above:

https://github.com/pandas-dev/pandas/pull/48272#issue-1352193730

Reply:
Why don't we just bump s3fs? The regular workflows are supposed be testing the latest dependencies anyways. It was green for me locally, so I'm going to try it in #48291.

Reply:
Don't really have a preference here, but in general we should not restrict the solver more than necessary regarding our direct dependencies

Reply:
Yeah, this looks like a mamba bug, since we're pulling a really old s3fs. It's weird too, since ``mamba repoquery whoneeds s3fs
``
gives me ``No entries matching "s3fs" found``